src/js/media-captions-button.js

```diff
@@ -155,4 +155,5 @@ if (!globalThis.customElements.get('media-captions-button')) {
   );
 }
 
+export { MediaCaptionsButton };
```

Comment:
Why is this needed?

Reply:
Plausibly not needed if we migrate to full TypeScript. The `@typedef` imports were getting confused from using the default module export. This is also a pattern in some other of our component modules. There are good reasons to do named exports/both (tl;dr - different build tools can get confused). Might be a good candidate for our development principles (in whatever direction we decide).